plugins/chinesehoroscope.py

```diff
@@ -0,0 +1,94 @@
+# Plugin by Infinity - <https://github.com/infinitylabs/UguuBot>
+
+import requests
+from bs4 import BeautifulSoup
+from sqlalchemy import Table, String, Column, select
+
+from cloudbot import hook
+from cloudbot.util import database
+
+table = Table(
+    'horoscope',
+    database.metadata,
+    Column('nick', String, primary_key=True),
+    Column('sign', String)
+)
+
+
+def get_sign(db, nick):
+    row = db.execute(select([table.c.sign]).where(table.c.nick == nick.lower())).fetchone()
+    if not row:
+        return None
+
+    return row[0]
+
+
+def set_sign(db, nick, sign):
+    res = db.execute(table.update().values(sign=sign.lower()).where(table.c.nick == nick.lower()))
+    if res.rowcount == 0:
+        db.execute(table.insert().values(nick=nick.lower(), sign=sign.lower()))
+
+    db.commit()
+
+
+@hook.command(autohelp=False)
+def horoscope(text, db, bot, nick, notice, notice_doc, reply, message):
```

Comment:
The command name is either derived from the function name or the first parameter to the `hook.command()` call, please update this command name to avoid conflicts